apps/client/pages/todo-list.tsx

```diff
@@ -69,7 +67,6 @@ const TodoList: React.FC = () => {
   if (error) {
     if (error.message.includes('Authentication failed')) {
       router.push('/login');
-      return <></>;
```

Comment:
I think this was needed to not continue rendering of the next things while pushing.

Reply:
will bring it back as I respect You very much @dimamik